docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.9"
+
+x-common-env: &config_env
+  PROP:
+
+services:
+  app:
+    image: kotlin-template:latest
+    container_name: kotlin-template
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8000:8000"
+    environment:
```

Comment:
Vou estar pondo aqui só pra poder dar concluir no comentário:

Acha que faz sentido usar as variáveis já?

Digo, no `SampleController.kt > sampleGet()`, algo como

```
        var primo: String
        primo = System.getenv("PROP")
        primo = if (primo != null) primo else ""

        var seccondo: String
        seccondo = System.getenv("PROP1")
        seccondo = if (seccondo != null) seccondo else ""

        return "Hello camelCase World! $primo || $seccondo"
```

Reply:
Detalhe (comentando pq pode não ser tão óbvio ou passa batido - até pq não tá na descrição do PR ahuahu)
- Se for feita a mudança, aparentemente precisa dar build de novo pra atualizar

Reply:
Mapeei esse lance de variáveis de ambiente pra KOT-24, que vai criar .env